app/assets/stylesheets/application.scss

```diff
@@ -103,3 +104,34 @@ main {
 .modal-backdrop {
   z-index: 10;        
 }
+
+.StripeElement {
```

Comment:
Is this actually kosher? Are these documented classes that Stripe says you should use to override their styling? Otherwise, they're prone to change with API changes, breaking your "hacked' styles

Reply:
Suggest moving them to some dedicated `stripe_overrides.scss` or something like that